src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -49,11 +49,15 @@ use crate::storage_value::StorageValue;
 use crate::store::*;
 use crate::StateStoreIter;
 
+const MAX_SPILL_TIMES: u64 = 2 << 16;
```

Comment:
Should be `1 << 16`? 

And if `16` is the value for shifting in epoch, we may define a const in the common crate and refer to the const there.